nixos/modules/services/networking/meshtastic.md

```diff
@@ -0,0 +1,51 @@
+# Meshtasticd {#module-services-meshtasticd}
+
+[Meshtasticd](https://meshtastic.org/) daemon.
+
+Meshtastic is an open-source, off-grid, decentralised mesh network designed to
+run on affordable, low-power devices.
+
+Meshtastic is a project that enables you to use inexpensive LoRa radios as a
+long range off-grid communication platform in areas without existing or reliable
+communications infrastructure. This project is 100% community driven and open
+source!
+
+## Quickstart {#module-services-meshtasticd-quickstart}
+
+A minimal configuration:
+
+```nix
+{
+  services.meshtasticd = {
+    enable = true;
+    port = 4403;
+    settings = {
+      Lora = {
+        Module = "auto";
+      };
+      Webserver = {
+        Port = 9443;
+        RootPath = pkgs.meshtastic-web;
+      };
+      General = {
+        MaxNodes = 200;
+        MaxMessageQueue = 100;
+        MACAddressSource = "eth0";
+      };
+    };
+  };
+}
+```
+
+This configuration starts the Meshtasticd server on port `4403` and the web
+interface on port `9443`.
```

Comment:
Does it listen on 0.0.0.0 or 127.0.0.1? Also a nginx with TLS might be nice especially if there is more to do than just these two things.